### DISCUSSION TITLE
Unfollowing People

### DISCUSSION THREAD

**Discussion by brxndonnn** [new92/igfi#24] [Q&A] [answered]
hi again, do i have to manually unfollow all the people that the script followed or does the script do it after a while?

**Comment by new92** [comment, marked as answer]
Hello ! Yeah the script unfollows the users but it may occasionally miss some. So don't worry, if you want to be 100% sure you can check and unfollow them manually. But I'm working on this right now so I hope it doesn't take me much time to fix. If you have any other questions or getting errors please do not hesitate to ask !

**Reply by brxndonnn** [reply]
awesome!
